### DISCUSSION TITLE
Cache file deleting.

### DISCUSSION THREAD

**Discussion by Zeyadas** [koreader/koreader#7285] [Q&A]
Am using an android tablet with the latest koreader version 1.1.2021. 
Can I delete the (cache) file? And what will happen if I did?
![Screenshot_2021-02-14-14-34-45](https://user-images.githubusercontent.com/70109540/107876746-df1ad100-6ed8-11eb-8242-3a8b5cd78008.png)

**Comment by poire-z** [comment]
You can (but make sure to properly exit KOReader, so it's not running in the background on Android).
It will be recreated.
EPUBs files you have previously opened will take a longer time to load on their first re-opening (as there's no longer a cache of what's been parsed and render).

**Reply by Zeyadas** [reply]
That was helpful. Thanks..

**Reply by NiLuJe** [reply]
Note that we have an in-KOReader setting to do that ;).

**Reply by Zeyadas** [reply]
The (in koreader) setting showed a ~2Mb of cache but the folder itself is more than 120Mb and my tablet is too old (8gb of internal memory including about 4gb for the android OS) so I'm used to this:
![20210215_181351](https://user-images.githubusercontent.com/70109540/107963784-a56dc700-6fb9-11eb-8880-e52b5bbeabcf.png)
I wonder if there's a way to (move) koreader and it's files to SD card.
